src/rest-server/deploy/auth-configmap/oidc.yaml.template

```diff
@@ -0,0 +1,32 @@
+{% if cluster_cfg['authentication']['OIDC'] %}
+
+{% if cluster_cfg['authentication']['OIDC-type'] == 'AAD' %}
+
+OIDC-type: aad
```

Comment:
do not need more config files, you can add those items as env variables in pod yaml (deploy/rest-server.yaml.template) directly.

Reply:
Whole file content will be a environment？

Reply:
Are all those values needed in rest server? Or this file is just a config file like `pai-configuration`?

It seems that this yaml is just a k8s configmap, so why do we need to place it here? Maybe other services will also need it?

Reply:
https://github.com/Microsoft/pai/blob/43a7f8e520775e515418cf68fb6ace0062bad535/src/rest-server/src/config/express.js#L62
See the example, all the value is needed.

Reply:
Other service won't have the permission to communicate with AAD.

Reply:
I see.